abr_control/controllers/osc.py

```diff
@@ -78,30 +77,46 @@ def generate(self, q, dq,
             current joint angles [radians]
         dq : float numpy.array
             current joint velocities [radians/second]
-        target_pos : float numpy.array
-            desired joint angles [radians]
-        target_vel : float numpy.array, optional (Default: numpy.zeros)
-            desired joint velocities [radians/sec]
+        target: 6 dimensional float numpy.array
+            desired task space position and orientation [meters, radians]
+        target_vel : float 6D numpy.array, optional (Default: None)
+            desired task space velocities [meters/sec, radians/sec]
+        ctrlr_dof : 6D list of boolean, optional (Default: position control)
+            specifies which task space degrees of freedom are to be controlled
+            [x, y, z, alpha, beta, gamma] (Default: [x, y, z]
+            NOTE: if more ctrlr_dof are specified than degrees of freedom in
```

Comment:
i was thinking about this and wasn't sure, because i think it'll depend also on the configuration of the robot. iirc it's related to the form of the Jacobian, another thing that will require more reading before implementation. i feeeeeel like having a note here is enough atm that we don't have to restrict the user ... not sure